docs/getting-started/ios/screenshots.md

```diff
@@ -90,7 +92,7 @@ XCUIApplication *app = [[XCUIApplication alloc] init];
 [Snapshot setupSnapshot:app];
 [app launch];
 ```
-
+1. In the terminal run `fastlane snapshot`.  WARNING: Running the test in Xcode will not create the snapshots is insufficient and will not generate the correct results - although no tests will fail.  The command line program creates the necessary subdirectories and renames the files as appropriate.
```

Comment:
Grammar needs some work here.

Reply:
Good catch, I was in a hurry.  Haste makes waste.